app/src/main/java/org/mozilla/fenix/tabstray/ext/Long.kt

```diff
@@ -0,0 +1,7 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.tabstray.ext
+
+internal fun Long?.orDefault() = this ?: -1L
```

Comment:
We forgot to write a doc that this "default" for the home fragment navigation.

Reply:
I'll add it.  Thanks